<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryReadingLineGridViewer</class>
 <widget class="QWidget" name="LibraryReadingLineGridViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>723</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>758</width>
    <height>723</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>758</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="LibraryReadingLineGridViewerCont" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="LongPressWidget" name="mainBookInfoCont" native="true">
        <property name="minimumSize">
         <size>
          <width>758</width>
          <height>327</height>
         </size>
        </property>
        <widget class="QWidget" name="currentBookCont" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>758</width>
           <height>339</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="coverCont" native="true">
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="margin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="2" column="1">
              <widget class="QWidget" name="coverTagsCont" native="true">
               <widget class="QLabel" name="titleLbl">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>53</width>
                  <height>17</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Rayuela</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignHCenter|Qt::AlignTop</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="indent">
                 <number>0</number>
                </property>
               </widget>
               <widget class="QLabel" name="authorLbl_2">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>92</width>
                  <height>17</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Julio cortazar</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignHCenter|Qt::AlignTop</set>
                </property>
                <property name="indent">
                 <number>0</number>
                </property>
               </widget>
               <widget class="QLabel" name="newTagLbl">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>140</width>
                  <height>218</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>140</width>
                  <height>218</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>140</width>
                  <height>218</height>
                 </size>
                </property>
                <property name="text">
                 <string>Nuevo</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignHCenter</set>
                </property>
                <property name="margin">
                 <number>13</number>
                </property>
                <property name="indent">
                 <number>0</number>
                </property>
               </widget>
               <widget class="DegreesLabel45" name="sampleTagLbl">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>173</width>
                  <height>85</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>173</width>
                  <height>85</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>173</width>
                  <height>85</height>
                 </size>
                </property>
                <property name="text">
                 <string>Muestra</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                </property>
                <property name="indent">
                 <number>0</number>
                </property>
               </widget>
               <widget class="QLabel" name="coverLbl">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>66</width>
                  <height>17</height>
                 </rect>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="DegreesLabel45" name="subscriptionTagLbl">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>173</width>
                  <height>85</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>173</width>
                  <height>85</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>173</width>
                  <height>85</height>
                 </size>
                </property>
                <property name="text">
                 <string>Suscripción</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                </property>
                <property name="indent">
                 <number>0</number>
                </property>
               </widget>
               <zorder>coverLbl</zorder>
               <zorder>titleLbl</zorder>
               <zorder>authorLbl_2</zorder>
               <zorder>newTagLbl</zorder>
               <zorder>sampleTagLbl</zorder>
               <zorder>subscriptionTagLbl</zorder>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="topRightLbl">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>24</width>
                 <height>14</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="bottomCenterLbl">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="topLeftLbl">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>14</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="topCenterLbl">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="bottomRightLbl">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>19</width>
                 <height>20</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="centerRightLbl">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>24</width>
                 <height>260</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="centerLeftLbl">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>260</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="bottomLeftLbl">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <spacer name="verticalSpacer_8">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="1">
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="0">
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0">
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1">
              <spacer name="verticalSpacer_7">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <spacer name="verticalSpacer_9">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
            <zorder>bottomCenterLbl</zorder>
            <zorder>coverTagsCont</zorder>
            <zorder>bottomLeftLbl</zorder>
            <zorder>bottomRightLbl</zorder>
            <zorder>topRightLbl</zorder>
            <zorder>centerLeftLbl</zorder>
            <zorder>centerRightLbl</zorder>
            <zorder>topLeftLbl</zorder>
            <zorder>topCenterLbl</zorder>
            <zorder>verticalSpacer_8</zorder>
            <zorder>verticalSpacer_5</zorder>
            <zorder>verticalSpacer_4</zorder>
            <zorder>verticalSpacer_6</zorder>
            <zorder>verticalSpacer_7</zorder>
            <zorder>verticalSpacer_9</zorder>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="currentBookData" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="currentBookData2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="readingLbl">
               <property name="text">
                <string>Leyendo...</string>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ElidedLabel" name="bookTitleLbl">
               <property name="text">
                <string>Las horas oscuras</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ElidedLabel" name="authorLbl">
               <property name="text">
                <string>Juan Francisco Ferrándiz</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="pagCont">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="readingPercentLbl">
                 <property name="text">
                  <string>40</string>
                 </property>
                 <property name="indent">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="percentLbl">
                 <property name="text">
                  <string>%</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="readLbl">
                 <property name="text">
                  <string>Leído</string>
                 </property>
                 <property name="indent">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="lastAccessCont">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lastAccessLbl">
                 <property name="text">
                  <string>Ultimo acceso:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="dateAccessLbl">
                 <property name="text">
                  <string>Fecha de acceso</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="LibraryLineGridViewerItem" name="book1" native="true"/>
      </item>
      <item>
       <widget class="LibraryLineGridViewerItem" name="book2" native="true"/>
      </item>
      <item>
       <widget class="LibraryLineGridViewerItem" name="book3" native="true"/>
      </item>
      <item>
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="LibraryPageHandler" name="VerticalPagerHandler" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LibraryPageHandler</class>
   <extends>QWidget</extends>
   <header>LibraryPageHandler.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElidedLabel</class>
   <extends>QLabel</extends>
   <header>ElidedLabel.h</header>
  </customwidget>
  <customwidget>
   <class>DegreesLabel45</class>
   <extends>QLabel</extends>
   <header>DegreesLabel45.h</header>
  </customwidget>
  <customwidget>
   <class>LongPressWidget</class>
   <extends>QWidget</extends>
   <header>LongPressWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LibraryLineGridViewerItem</class>
   <extends>QWidget</extends>
   <header location="global">LibraryLineGridViewerItem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
